components/GraphDiagram/GraphDiagram.scss

```diff
@@ -95,20 +96,24 @@ g.node {
 
   &.selected {
     & > path {
-      stroke-width: 3;
-      stroke: var(--highlight) !important;
+      fill: var(--accent) !important;
+      stroke: var(--accent) !important;
+      stroke-width: 5;
+    }
+    & > text {
+      fill: var(--text-on-accent);
     }
   }
 
   &.upstream {
     & > path {
-      stroke: var(--stroke-blue);
+      stroke: var(--accent-100);
```

Comment:
Dependents: Same as accent, but lighter:

Reply:
Can we make the distinction between upstream vs. downstream dependencies more obvious?  It's pretty subtle at the moment and I've seen graphs with circular dependencies where being able to tell the difference is really helpful.  (Sorry, can't seem to find a good example at the moment.)

Seeing that difference was the initial motivation for the blue-violet colors.  Although, admittedly, I chose those colors while developing the feature just to make them *blatantly* obvious, then never got around to finessing them so... yeah... I totally understand your reaction.

Maybe use slightly different hues of orange?   E.g. make `upstream` be `color-mix(in hsl shorter hue, var(--accent) 25%, var(--stroke-yellow))`?

Reply:
As a reaction to huge dependency trees caused by a certain user, people have been posting npmgraph screenshots of the before/after tree cleanup. It'd be nice to have those trees look good, that's why I wanted to remove the cyan/magenta colors.

I'll run some tests and see how to generate an appealing palette from the accent.

Reply:
By the way, do you have an example of a tree with circular dependencies? It would be good to have a list of such dependencies to test out certain features. They could be added to the contributing.md file.

Reply:
Alas, not at the moment.  I'll update here the next time I see one.  I typically see them in projects (as opposed to published modules) where plugins and dependencies tend to be a bit messier.

Reply:
Here's a [somewhat contrived] example: https://npmgraph.js.org/?q=string-width%404.2.3%2C+yargs%4016%2C+wrap-ansi%407%2C+#select=exact%3Acliui%407.0.4

Notice how upstream/downstream nodes and edges kind of blend together here ...

![CleanShot 2024-07-22 at 09 59 52](https://github.com/user-attachments/assets/3c7d91c1-2398-4809-b92e-ed00a4782189)

... versus what's currently on `main` (which is admittedly a bit over-the-top colorwise :-) )

![CleanShot 2024-07-22 at 10 00 36](https://github.com/user-attachments/assets/3c8d9391-2823-4c66-8781-6353df8a5789)